exercises/diffie-hellman/diffie-hellman.spec.js

```diff
@@ -0,0 +1,67 @@
+import DiffieHellman from './diffie-hellman';
+
+describe('diffie-hellman', () => {
+  const p = 23;
+  const g = 5;
+  let diffieHellman = new DiffieHellman(p, g);
+
+  const alicePrivateKey = 6;
+  const alicePublicKey = 8;
+
+  const bobPrivateKey = 15;
+  const bobPublicKey = 19;
+
+  test('throws an error if the constructor arguments are out of range', () => {
+
+    expect(() => {
+      new DiffieHellman(0, 9999)
+    }).toThrow();
+
+  });
+
+  test('throws an error if the constructor arguments are not prime', () => {
+
+    expect(() => {
+      new DiffieHellman(10, 13)
+    }).toThrow();
+
+  });
+
+  test('throws an error if private key >= p-1', () => {
```

Comment:
@rchavarria I took a look at the other existing test suites (C#, F#, Elixir, Go) and they all just have one test for the range, similar to the one I wrote.

Would you be OK with just adding two more tests or do you feel that all six should be added?